### DISCUSSION TITLE
Mac M1  install

### DISCUSSION THREAD

**Discussion by cperiodj3** [thygate/stable-diffusion-webui-depthmap-script#142] [General]
Any idea why this doesnt work on a Ventura Mac M1 Install? I installed via Extensions>Install from URL but i depth doesnt appear on the interface. I dont get any errors, depth just doesnt appear. Any help would be highly appreciated.

**Comment by thygate** [comment]
I just merged a pull request that changes to qt6 for 'Darwin' systems, it might solve this issue.
Check the console for error messages..

**Reply by cperiodj3** [reply]
reading through some of the issues i saw that its possible that ffmpeg may not be installed. i saw that there was an error for this in my terminal so i ran brew install ffmpeg. Now i see depth in the ui. Thanks. now i am seeing this error when i choose to generated 4 video demos 

RuntimeError: Could not import backend "EGL": dlopen(EGL.framework/EGL, 0x0006): tried: 'EGL.framework/EGL' (no such file), '/System/Volumes/Preboot/Cryptexes/OSEGL.framework/EGL' (no such file), '/opt/homebrew/lib/EGL.framework/EGL' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/opt/homebrew/lib/EGL.framework/EGL' (no such file), '/usr/lib/EGL.framework/EGL' (no such file, not in dyld cache), 'EGL.framework/EGL' (no such file), '/Library/Frameworks/EGL.framework/EGL' (no such file), '/System/Library/Frameworks/EGL.framework/EGL' (no such file, not in dyld cache)

**Reply by cperiodj3** [reply]
If i dont check the box to generate 4 videos, i am able to successfully create a depth map which is cool but i would really like for the video piece to work as well...if it can

**Reply by thygate** [reply]
> reading through some of the issues i saw that its possible that ffmpeg may not be installed. i saw that there was an error for this in my terminal so i ran brew install ffmpeg. Now i see depth in the ui. Thanks. now i am seeing this error when i choose to generated 4 video demos
> 
> RuntimeError: Could not import backend "EGL": dlopen(EGL.framework/EGL, 0x0006): tried: 'EGL.framework/EGL' (no such file), '/System/Volumes/Preboot/Cryptexes/OSEGL.framework/EGL' (no such file), '/opt/homebrew/lib/EGL.framework/EGL' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/opt/homebrew/lib/EGL.framework/EGL' (no such file), '/usr/lib/EGL.framework/EGL' (no such file, not in dyld cache), 'EGL.framework/EGL' (no such file), '/Library/Frameworks/EGL.framework/EGL' (no such file), '/System/Library/Frameworks/EGL.framework/EGL' (no such file, not in dyld cache)

Please update the extension and restart the backend, it should use qt6 on darwin now, not egl.

**Reply by cperiodj3** [reply]
"Please update the extension and restart the backend" Does restart the backend mean apply and restart ui or webui.sh? After updating the extension (latest), apply and restart ui, i now get the red error without an actual error message.

**Reply by cperiodj3** [reply]
i think i understand now. i have to install pyqt6 using pip install pyqt6 and all appears to be well. creating depth maps and videos now. i appreciate the help.
